### PR TITLE
[MOD]end2end_baseline_fixed_dataset

### PR DESCRIPTION
**Pull request by cyh5757**
from_generator로 시도 실패. .from_tensor_slices로 변경
tf.data.Dataset.from_tensor_slices((train_dataset.iloc[:,0:3])) 오류. 
(train_dataset['image_path'].values,train_dataset['Gender'].values ...) 이렇게 변경

suffled_fold_dataset을 image_dataset로
하지만, 학습이 제대로 안됨. gradientTape을 수정 예정.

### CODE REVIEW

**Comment by cyh5757** [comment]
- [x] gradientTape 수정
- [x] 데이터가 다 안들어가는 문제가 있음. ->데이터는 잘들어가고 있고, 학습이 잘 안되는 중 -> 모델 layer 추가

**Comment by cyh5757** [comment]
baseline 학습이 안돌아가서 여러 방법으로 모델 변경.
Accuracy: 46%-77%까지 성능 향상.
Loss : 0.702--->0.478 향상

우선 baseline이 완벽한 paper implementation은 아님.
다시 baseline을 완벽히 구현후, 재학습 예정
